Samples/word-citation-management/src/taskpane/taskpane.html

```diff
@@ -24,6 +24,9 @@
 </head>
 
 <body class="ms-font-m ms-welcome ms-Fabric ms-bgColor-white">
+    <!-- The following image URL tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project. -->
+    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/word-citation-management-run" />
```

Comment:
If we didn't have this, how were we getting sample run data?
(Perhaps a question for @davidchesnut)

Reply:
Perhaps. I did have telemetry on the readme.